app/views/admin/configs/show.html.erb

```diff
@@ -1,952 +1,928 @@
-      data-target="#siteConfigBodyContainer" aria-expanded="false" aria-controls="siteConfigBodyContainer">
-      <h2 class="d-inline">All Site Configuration</h2>
-      <button class="btn btn-secondary float-right" type="button">
-        Toggle Site Configuration
-      </button>
-    </div>
-    <div id="siteConfigBodyContainer" class="collapse hide hide p-3" aria-labelledby="siteConfigHeader">
-      <%= form_for(SiteConfig.new, url: admin_config_path) do |f| %>
-
-        <div class="card mt-3">
-          <%= render partial: "card_header",
-                     locals: {
-                       header: "API Tokens",
-                       state: "collapse",
-                       target: "apiBodyContainer",
-                       expanded: "false"
-                     } %>
-          <div id="apiBodyContainer" class="card-body collapse hide" aria-labelledby="apiBodyContainer">
-            <div class="form-group">
-              <%= admin_config_label :health_check_token %>
-              <%= f.text_field :health_check_token,
-                               class: "form-control",
-                               value: SiteConfig.health_check_token,
-                               placeholder: Constants::SiteConfig::DETAILS[:health_check_token][:placeholder] %>
-              <div class="alert alert-info"><%= Constants::SiteConfig::DETAILS[:health_check_token][:description] %></div>
-            </div>
-          </div>
-        </div>
-
-        <div class="card mt-3">
-          <%= render partial: "card_header",
-                     locals: {
-                       header: "Authentication",
-                       state: "collapse",
-                       target: "authenticationBodyContainer",
-                       expanded: "false"
-                     } %>
-          <div id="authenticationBodyContainer" class="card-body collapse hide" aria-labelledby="authenticationBodyContainer">
-            <div class="form-group">
-              <%= admin_config_label :authentication_providers %>
-              <%= select_tag "site_config[authentication_providers]",
-                             options_from_collection_for_select(
-                               authentication_available_providers,
-                               :provider_name,
-                               :official_name,
-                               authentication_enabled_providers.map(&:provider_name),
-                             ),
-                             multiple: true,
-                             class: "form-control selectpicker" %>
-              <div class="alert alert-info"><%= Constants::SiteConfig::DETAILS[:authentication_providers][:description] %></div>
-            </div>
-            <div class="notice">
-              Notice: Changing authentication keys will not take affect until devops restarts the app (about once per day). This will soon be instant.
-            </div>
```

Comment:
@ludwiczakpawel looks like this may have been reverted back to an <h4> when you merged, hence I was not seeing the relevant styles being applied. 

Can you please revert to this, and maybe change notice to .crayons-notice as you mention [here]([https://github.com/forem/forem/pull/10015#discussion_r477358925])

Reply:
Done.